plugin/storage/grpc/shared/grpc_client.go

```diff
@@ -72,14 +67,6 @@ func (c *GRPCClient) StreamingSpanWriter() spanstore.Writer {
 	return newStreamingSpanWriter(c.streamWriterClient)
 }
 
-func (c *GRPCClient) ArchiveSpanReader() spanstore.Reader {
-	return &archiveReader{client: c.archiveReaderClient}
```

Comment:
are we deleting these internal archiveReader/archiveWriter types?

Reply:
removed